### DISCUSSION TITLE
shapely库不能下载

### DISCUSSION THREAD

**Discussion by monkeycc** [PaddlePaddle/PaddleOCR#13888] [Q&A]
https://paddlepaddle.github.io/PaddleOCR/ppocr/quick_start.html#12-paddleocr-whl
对于Windows环境用户：直接通过pip安装的shapely库可能出现[winRrror 126] 找不到指定模块的问题。建议从[这里](https://www.lfd.uci.edu/~gohlke/pythonlibs/#shapely)下载shapely安装包完成安装。

提供的地址 不能下载

百度是差这点空间吗
怎么不放百度云

**Comment by UserWangZz** [comment]
大哥这也能喷啊，pip安装建议换源，国外网站不稳定
